metadata-ws/src/test/java/uk/ac/ebi/ampt2d/metadata/cucumber/CommonSteps.java

```diff
@@ -108,29 +118,47 @@ public void cleanStates() {
 
     @When("^I request GET ([\\S]*)$")
     public void performGetOnResourceUri(String resourceUri) throws Exception {
+        CommonStates.setResultActions(mockMvc.perform(get(resourceUri)
+                .with(CommonStates.getRequestPostProcessor()))
+        );
+    }
+
+    @When("^I request anonymous GET ([\\S]*)$")
+    public void performGetOnResourceUriNonSecure(String resourceUri) throws Exception {
         CommonStates.setResultActions(mockMvc.perform(get(resourceUri)));
     }
 
     @When("^I request GET with value of (.*)$")
     public void performGetWithResourceUriKey(String resourceUriKey) throws Exception {
-        CommonStates.setResultActions(mockMvc.perform(get(CommonStates.getUrl(resourceUriKey))));
+        CommonStates.setResultActions(mockMvc.perform(get(CommonStates.getUrl(resourceUriKey))
+                .with(CommonStates.getRequestPostProcessor())));
     }
 
     @When("^I request GET for (.*) of (.*)$")
     public void performGetForLinkedObjects(String className, String resourceUriKey) throws Exception {
-        CommonStates.setResultActions(mockMvc.perform(get(CommonStates.getUrl(resourceUriKey) + "/" + className)));
+        CommonStates.setResultActions(mockMvc.perform(get(CommonStates.getUrl(resourceUriKey) + "/" + className)
+                .with(CommonStates.getRequestPostProcessor())));
     }
 
     @When("^I request POST (.*) with JSON payload:$")
     public void performPostOnResourceUriWithJsonData(String resourceUri, String jsonData) throws Exception {
         CommonStates.setResultActions(mockMvc.perform(post(resourceUri)
+                .with(CommonStates.getRequestPostProcessor())
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(jsonData.getBytes())));
+    }
+
+    @When("^I request PUT (.*) with JSON payload:$")
+    public void performPutOnResourceUriWithJsonData(String resourceUrlKey, String jsonData) throws Exception {
+        CommonStates.setResultActions(mockMvc.perform(put(CommonStates.getUrl(resourceUrlKey))
+                .with(CommonStates.getRequestPostProcessor())
                 .contentType(MediaType.APPLICATION_JSON)
                 .content(jsonData.getBytes())));
     }
 
     @When("^I request PATCH (.*) with list (.*) of (.*)")
     public void performPatchOnResourceWithLinkedObjectList(String urlKey, String linkedObjectUrlKeys,
-                                                           String linkedObjectClassName) throws Exception {
+                                                       String linkedObjectClassName) throws Exception {
```

Comment:
Restore previous indentation.

Reply:
yes, current code has `(` and `i` at column 59

Reply:
For me the indentation is still broken in the latest version of this PR; `String linkedObjectClassName` starts on column 56 instead of 60

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/57/commits/96df8efb9cb44ab559b908a49b25d1ca6e9df628